public/app/plugins/datasource/loki/components/LokiQueryField.tsx

```diff
@@ -63,7 +63,6 @@ function willApplySuggestion(suggestion: string, { typeaheadContext, typeaheadTe
 
 export interface LokiQueryFieldProps extends ExploreQueryFieldProps<LokiDatasource, LokiQuery, LokiOptions> {
   history: LokiHistoryItem[];
-  absoluteRange: AbsoluteTimeRange;
```

Comment:
@ivanahuckova @zoltanbedi I could not find anything consuming this field. Have I totally missed something?

Reply:
We used to use it for cascader button that showed label + values options. In 7.5 we switched to LabelBrowser that doesn't use it. So it is okay to remove. 🙂

Reply:
Thanks 🙏🏻